src/Sparrow/Json/IBlittableJsonTextWriter.cs

```diff
@@ -24,6 +24,7 @@ public interface IBlittableJsonTextWriter
     void WriteBool(bool val);
     void WritePropertyName(ReadOnlySpan<byte> prop);
 
+    void WritePropertyName(LazyStringValue prop);
     void WritePropertyName(string prop);
```

Comment:
It makes sense for us to push this change to the v7.0 codebase to avoid divergencies at the Blittable interface. cc @arekpalinski

Reply:
I looks that changes made here got reverted as there is nothing changed here